packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -14,18 +14,25 @@
  * limitations under the License.
  */
 
-import {BasePlugin, CanonicalCode, Func, HeaderGetter, HeaderSetter, MessageEventType, RootSpan, Span, SpanKind, TraceOptions} from '@opencensus/core';
+import {BasePlugin, CanonicalCode, Func, HeaderGetter, HeaderSetter, MessageEventType, Span, SpanKind, TagMap, TraceOptions} from '@opencensus/core';
 import * as httpModule from 'http';
 import * as semver from 'semver';
 import * as shimmer from 'shimmer';
 import * as url from 'url';
 import * as uuid from 'uuid';
-import {HttpPluginConfig, IgnoreMatcher} from './types';
+import * as stats from './http-stats';
+import {IgnoreMatcher} from './types';
 
 export type HttpGetCallback = (res: httpModule.IncomingMessage) => void;
 export type HttpModule = typeof httpModule;
 export type RequestFunction = typeof httpModule.request;
 
+// tslint:disable-next-line:no-any
+function isOpenCensusRequest(options: any) {
```

Comment:
What would you think about making a simple interface for this rather than passing in `any` here? Seems like it just has an optional `headers` attribute that is a map of string to string.

Reply:
Changed to ```RequestOptions```, instead of ```any```.